### DISCUSSION TITLE
Ghost torrents added to transfer list

### DISCUSSION THREAD

**Discussion by KingdomDesignsConsulting** [transmission/transmission#5267] [Support] [answered]
Since updating to the newest version of Transmission 4.0.2 (2a57b17031) on MacOS, a number of removed torrents keep re-adding themselves to my transfer list. They weren't in the list before updating and when I remove them, they re-appear the next time I relaunch the app. Is there some way to force their removal or is it just a programming bug? I'm guessing there is old some hidden file that keeps getting reloaded on relaunch.

**Comment by ckerr** [comment, marked as answer]
Hey @KingdomDesignsConsulting, thanks for asking.

Check out `/home/yourusername/Library/Application Support/Transmission/torrents` and see if there are some old files there that for whatever reason aren't getting removed.

- One possibility is they got marked read-only somewhere along the line and so Transmission tries but fails to remove them & they keep coming back when you restart. If so, just manually fix the permissions + remove them.

- Another possibility is it's a filename that Transmission doesn't expect. When you remove a torrent from Transmission, the code looks in that folder to remove files of the form `${name}.${infohash.substr(0, 16)}.torrent` (the old naming scheme, which was replaced because it can hit "filename too long" errors for long names) and `${infohash}.torrent` (the newer naming scheme).

If you see files in that folder that don't match those patterns, please LMK so that whatever other case you're hitting can be handled in 4.0.3 :smile_cat:

**Reply by zooeybear** [reply]
I'm having the issue as well. There's no data associated, and every time I try to remove it manually, it reappears when I reopen Transmission.

**Comment by KingdomDesignsConsulting** [comment]
That first one fixed it, thanks.

**Comment by cassepipe** [comment]
What solved it for me was to delete the `$XDG_CONFIG_HOME/transmission/resume` folder

After that I had to Ctrl + A all of my torrents and "Verify Local Data" but that solved it for me
